sdk/include/crt/math.h

```diff
@@ -235,6 +235,7 @@ _Check_return_ __CRT_INLINE float sinhf(_In_ float x) { return (float)sinh((doub
 _Check_return_ __CRT_INLINE float sqrtf(_In_ float x) { return (float)sqrt((double)x); }
 _Check_return_ __CRT_INLINE float tanf(_In_ float x) { return (float)tan((double)x); }
 _Check_return_ __CRT_INLINE float tanhf(_In_ float x) { return (float)tanh((double)x); }
+_Check_return_ __CRT_INLINE float exp2f(_In_ float x) { return (float)expf(x * logf(2)); }
```

Comment:
Could you maybe move this one just after the `expf` function (line 226), line it's done for these other "2f" (and 10f") versions?